### PR TITLE
Uses new Service to show availability badge (#145).

### PR DESCRIPTION
**Pull request by bwatson78**
- .rubocop.yml: excludes the view spec file from 2 rubocop rules.
- app/views/catalog/_show_header.html.erb: brings in this partial so that badge JS and span can be added.
- spec/fixtures/alma_availability_test_file_*: creates additional test XMLs that produce different results.
- spec/spec_helper.rb: further configures Webmock so that API calls can be stubbed with whatever we want.
- spec/system/view_show_page_spec.rb: inserts tests for the presence of all three options.

### CODE REVIEW

**Comment by bwatson78** [comment]
Screen Shot 2021-01-22 at 5 00 21 PM
Screen Shot 2021-01-22 at 5 00 49 PM
Screen Shot 2021-01-22 at 5 01 30 PM

File: app/views/catalog/_show_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    dataType: 'json',
+    success: function(data) { 
+      var availabilityHtml = '';
+      var physicalExists = data['<%= document.id %>']['physical']['exists'];
+      var physicalAvailable = data['<%= document.id %>']['physical']['available'];  
+
+      if (physicalExists == true && physicalAvailable == true) {
+        availabilityHtml = '<span class="badge badge-success">Available</span>';
+      } else if (physicalExists == true && physicalAvailable == false) {
+        availabilityHtml = '<span class="badge badge-danger">UnAvailable</span>';
```
Comment (alexBLR) [inline app/views/catalog/_show_header.html.erb:19]: Is there a reason why "A" is capitalized in unavailable?

**Review by alexBLR — CHANGES_REQUESTED** [review]
Just a small comment about capitalizing "A" in unavailable

File: app/views/catalog/_show_header.html.erb (hunk 1 above)
Comment (bwatson78) [reply app/views/catalog/_show_header.html.erb:19]: I have no clue why I did that. Some weird artistic flourish of mine? Anyways, I'll fix it.

Comment (bwatson78) [reply app/views/catalog/_show_header.html.erb]: A new commit has been pushed with the corrections.

**Comment by bwatson78** [comment]
Screen Shot 2021-01-25 at 9 19 00 AM

**Review by alexBLR — APPROVED** [review]
👍

**Pull request merged**